CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Welcome to Fleet Telemetry contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project!
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
+## Getting started
+
+### How to contribute ?
+We will accept contributions from anybody assuming they respect our policies and help improve the platform. here's the different type of contribution we support.
+
+#### :lady_beetle: Issues
+[Issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues) are used to track tasks that contributors can help with. If an issue has a triage label, we haven't reviewed it yet, and you shouldn't begin work on it.
+
+If you've found something in the code that should be updated, search open issues to see if someone else has reported the same thing. If it's something new, open an issue. We'll use the issue to have a conversation about the problem you are having or want to fix.
+
+#### :hammer_and_wrench: Pull requests
+A [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) is a way to suggest changes in our repository. When we merge those changes, it will kick of a new build. Currently we will manually push the update to docker to ensure a good understanding of release should be a major, minor or patch. We might automate this process in the future.
+
+
+#### :question: Support
+We are a small team working hard to keep up with the documentation demands of a continuously changing product. Unfortunately, we can't help with support questions in this repository. If you are experiencing a problem unrelated to the code hosted here please [contact Tesla Support directly](https://https://www.tesla.com/support). Any issues, discussions, or pull requests opened here requesting support will be given information about how to contact Tesla Support, then closed and locked.
+
+#### :earth_asia: Translations
+
+This website is internationalized and available in multiple languages. The source content in this repository is written in English.
```

Comment:
is this accurate? If yes, can we post link for translated content in the documentation for quick access?

Reply:
This website is internationalized and available => github. The source content in this repository is written in English => fleet streaming